src/components/profiles/messages/ChatHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button } from '@mui/material';
+
+interface ChatHeaderProps {
+  selectedContactUUID: string;
+  selectedContactFirstName: string;
+  selectedContactLastName: string;
+}
+
+export function ChatHeader({
+  selectedContactFirstName,
+  selectedContactLastName,
+  selectedContactUUID,
+}: ChatHeaderProps) {
+  const handleClick = () => {
+    alert('Navigate to ' + selectedContactUUID);
+  };
+
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'flex',
+          width: '100%',
+          height: '7.5%',
```

Comment:
I would recommend avoid using hard coded height values, specifically in %. I would recommend use padding/margin composition instead